src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -184,6 +186,11 @@ public Claim getClaimByExternalId(String externalId, User user) {
         return caseRepository.getClaimsByState(claimState, user);
     }
 
+    public InitiatePaymentResponse initiatePayment(String authorisation, String submitterId, InitiatePaymentRequest request) {
+        User user = userService.getUser(authorisation);
+        return caseRepository.initiatePayment(user, submitterId, request);
+    }
```

Comment:
FMI: Will the payhub client be invoked before or after this?

Reply:
after, the pay hub client will be invoked in the callback, ie. https://github.com/hmcts/cmc-claim-store/pull/1124/files#diff-ef12ef8695ae4d3fe705d8258e9b382bR39